### PR TITLE
With CORS

### PR DESCRIPTION
**Pull request by linktohack**
Simple is the best but should it be more useful with some CORS headers...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,12 +12,21 @@
...
+    "open graph",
+    "opengraph",
+    "ogp"
+  ],
   "author": "Bram Borggreve (github:beeman twitter:beeman_nl)",
   "license": "MIT",
   "dependencies": {
     "micro": "^7.3.3",
     "microrouter": "^2.1.0",
+    "now": "^8.5.2",
```
Comment (jacktuck) [inline package.json]: not sure now is needed as a dep there?

Comment (linktohack) [reply package.json]: You're right. Of course not, I will rebase if @beeman is interested in the PR.

Comment (beeman) [reply package.json]: @linktohack I am interested - if you can change the PR to only include the change in index.js I'll merge it today!

Comment (beeman) [reply package.json]: It makes a lot of sense to have CORS enabled here.

Comment (linktohack) [reply package.json]: PR updated, by CORS, do you mean [this](https://github.com/possibilities/micro-cors)?

**Comment by beeman** [comment]
Thanks @linktohack!

**Pull request merged**